kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -164,7 +164,7 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
   }
 
   def getBatchSessionImpl(sessionHandle: SessionHandle): KyuubiBatchSessionImpl = {
-    getSession(sessionHandle).asInstanceOf[KyuubiBatchSessionImpl]
+    Option(handleToSession.get(sessionHandle)).map(_.asInstanceOf[KyuubiBatchSessionImpl]).orNull
```

Comment:
we can add a new method `getSessionOption` in parent class instead of exposing `handleToSession`.

Reply:
thanks